jenkins_test.go

```diff
@@ -212,7 +212,7 @@ func TestCopyDeleteJob(t *testing.T) {
 func TestGetPlugins(t *testing.T) {
 	ctx := context.Background()
 	plugins, _ := jenkins.GetPlugins(ctx, 3)
-	assert.Equal(t, 10, plugins.Count())
+	assert.Greater(t, plugins.Count(), 5)
```

Comment:
I think this should be just 1, assuming when run this test on different servers, they all has at least 1 plugin installed.

Reply:
@figo Done